### PR TITLE
Users/kaadhina/mergetestplan

### PR DESCRIPTION
**Pull request by kaadhina**
Bug 521411
Merging the execution options in RunFunctionalTest task
Users of run functional test task can now execute tests from test plan.
The feature is under preview right now, and has been validated by hosted customers.
We are now enabling this for all customers.

Is the fix already in master?  No. Bug 536817
What testing was done?
         Manual  validation of E2E scenarios for CI & CD, Ran UTs
         Compat validation for existing build definitions using test assembly
Does this fix change the AT/DT contract? No
Does this fix include any database servicing changes? No

### CODE REVIEW

File: Tasks/RunDistributedTests/Strings/resources.resjson/en-US/resources.resjson [hunk 1]
```diff
@@ -5,12 +5,19 @@
   "loc.instanceNameFormat": "Run Tests $(sourcefilters) on $(testMachineGroup)",
   "loc.group.displayName.setupOptions": "Setup Options",
   "loc.group.displayName.executionOptions": "Execution Options",
-  "loc.group.displayName.runUsingTestPlan": "Execution Options - Run using Test Plan",
   "loc.group.displayName.reportingOptions": "Reporting Options",
   "loc.input.label.testMachineGroup": "Test Machine Group / Azure Resource Group",
   "loc.input.help.testMachineGroup": "Name of the Test Machine Group or Azure Resource Group on which tests need to be run.",
   "loc.input.label.dropLocation": "Test Drop Location",
   "loc.input.help.dropLocation": "Location where the test binaries have been dropped in the agent machine(s) as part of the 'Windows Machine File Copy' or 'Azure File Copy' task. System Environment Variables can also be used in location string. e.g., `%systemdrive%\\Tests`, `%temp%\\DropLocation` etc.",
+  "loc.input.label.testSelection": "Test Selection",
```
Comment (acesiddhu) [inline Tasks/RunDistributedTests/Strings/resources.resjson/en-US/resources.resjson]: do we know who will update these strings for us for update 2 RTW?

**Comment by acesiddhu** [comment]
any other fixes we did on vsts which we want to migrate to update 2 branch?

**Comment by acesiddhu** [comment]
for testing lets do these as well
1) Non test plan scenarios in build and RM
2) run our e2e test as well

**Comment by kaadhina** [comment]
#1 is done. #2 Will need test fix since we are introducing a new field now. Will do that in master

**Comment by acesiddhu** [comment]
and test plan thing tested on RM

**Comment by kaadhina** [comment]
Yeah, both test plan, test assembly, tested on both build and RM

**Comment by acesiddhu** [comment]
rest looks fine to me. Get this reviewed by Satish as well

**Comment by acesiddhu** [comment]
we need to enable test suite filter in summary page by default for update2

**Comment by SatishThareja** [comment]
the changes look fine.

File: Tasks/RunDistributedTests/Strings/resources.resjson/en-US/resources.resjson (hunk 1 above)
Comment (kaadhina) [inline Tasks/RunDistributedTests/Strings/resources.resjson/en-US/resources.resjson]: I'm syncing with Eric for this.

**Pull request merged**